src/uniprotkb/components/entry/FunctionSection.tsx

```diff
@@ -376,6 +376,13 @@ const FunctionSection = ({
           />
         </Suspense>
       </ErrorBoundary>
+      <ErrorBoundary>
+        <Tabs>
```

Comment:
I think you have Tabs here, but also again within GoCam where you have Tabs and a Tab